.orchestra/ci/install-old-system-dependencies.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Move this as a comment in the `qt` component.

Did you really think a file with `old` in the name was going to pass code review? `:D`

Reply:
Fixed